src/playground/public/assets/js/app.ts

```diff
@@ -81,6 +87,7 @@ async function executeAskScript(
     const json = await response.json();
     if (response.status == 200) {
       showSuccessfulResponse(json.result);
+      playgroundIsDirty(false);
```

Comment:
```suggestion
      isDirty = false;
```

Reply:
what about this isDirty flag? Now is undefined variable for this scope. If I push dirty params to all functions eg. 
```
async function executeAskScript(
  askScriptCode: string,
  askScriptServerUrl: string,
  isDirty: boolean
) {
```
`executeAskScriptFromEditor(editor, askScriptServerUrl, isDirty); ` the flag dosnt update in the top.